app/lib/screens/mess/student/mess_student.dart

```diff
@@ -168,3 +166,102 @@ class _MessStudentScreenState extends State<MessStudentScreen> {
     );
   }
 }
+    // return FutureBuilder(
+    //     future: fetchMessData(),
+    //     builder: (BuildContext context,
+    //         AsyncSnapshot<Map<String, List<Mess>>> snapshot) {
```

Comment:
Make sure to remove residual code before asking for review

Reply:
removed